broker-core/src/main/java/io/zeebe/broker/subscription/message/processor/MessageEventProcessors.java

```diff
@@ -46,11 +46,7 @@ public static void addMessageProcessors(
         .onCommand(
             ValueType.MESSAGE,
             MessageIntent.PUBLISH,
-            new PublishMessageProcessor(
-                messageState,
-                subscriptionState,
-                startEventSubscriptionState,
-                subscriptionCommandSender))
+            new PublishMessageProcessor(zeebeState, subscriptionCommandSender))
```

Comment:
I'd rather we pass in only the direct dependencies as opposed to passing the whole parent object; it makes it clearer what the dependencies of the object are, and also makes it a whole lot easier to test/reuse in isolation.

I can do this change myself though, so no worries.